runtime/compiler/control/CompilationThread.cpp

```diff
@@ -2711,6 +2711,9 @@ void TR::CompilationInfo::stopCompilationThreads()
 
       if (getJProfilerThread())
          fprintf(stderr, "Allocated memory for profile info = %d KB\n", getJProfilerThread()->getProfileInfoFootprint()/1024);
+
+      if (trPersistentMemory)
```

Comment:
These statistics are probably useful only when tracking a memory leak. Also, the list of memory objects is very long.
For these two reasons I would like the printout to be enabled by another environment variable.